Brokkr/src/main/cpp/subsystems/Arm.cpp

```diff
@@ -42,6 +44,11 @@ double Arm::SetArmHeight(double pose)
     return pose;
 }
 
+double Arm::CANCoderArmStatus()
```

Comment:
Seems like `CANCoderArmStatus` is really `Position`.

Why are we getting position two different ways? Is the ArmMotor sensor position still useful? Should one of these be removed?

Reply:
Work out the answer later.